server/historian/package.json

```diff
@@ -44,7 +44,7 @@
     "temp-directory": "nyc/.nyc_output"
   },
   "devDependencies": {
-    "@fluidframework/build-tools": "^0.2.43260",
+    "@fluidframework/build-tools": "^0.2.43369",
```

Comment:
Does the historian package-lock.json also need to be updated?

Reply:
Good catch, thanks

Reply:
@hedasilv - The Historian build passed w/o package-lock change, it looks like the pipeline does `npm i` not `npm ci` (It uses the docker container setup).  Should I open an issue here, or is this the desired configuration for CI?

https://dev.azure.com/fluidframework/public/_build/results?buildId=43420&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=440a145c-148e-58c1-7881-71912bd6595e&l=842

Reply:
Hmmmm good question. It looks like the routerlicious pipeline is doing the same as Historian: https://dev.azure.com/fluidframework/public/_build/results?buildId=43431&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=440a145c-148e-58c1-7881-71912bd6595e&l=884

The behavior seems consistent with r11s so maybe that's ok? Also, package-lock.json would get eventually updated (and I'm guessing when we run things in Docker, it might get updated thanks to the npm install that we run there). It doesn't feel like something that should break the pipeline (unless I'm missing something important around npm and package-lock.json). @znewton Do you have any opinions on this?